packages/forklift-console-plugin/src/utils/namespaces.ts

```diff
@@ -0,0 +1,17 @@
+import { Namespace } from './constants';
+import { isUpstream } from './env';
+
+/**
+ * When using an upstream build, use the env default namespace
+ * with a fallback of 'default', otherwise use 'openshift-mtv'.
+ * @returns string
+ */
+export const getDefaultNamespace = (): string => {
+  const isUserUpstream = isUpstream();
+
+  if (isUserUpstream) {
+    return process.env.DEFAULT_NAMESPACE || Namespace.Default;
```

Comment:
if its upstream should it be Namespace.KonveyorForklift ?
why are we looking on process.env.default_namespace?

Reply:
The JIRA description mentioned adding a condition for this env variable so that's why I tried to still use it here, but yes, I can just update to Namespace.KonveyorForklift instead to make it more straightforward.